lib/sidekiq_bus/middleware/retry.rb

```diff
@@ -0,0 +1,29 @@
+module SidekiqBus
+  module Middleware
+    module Server
+      
+      # ensure sidekiq will retry jobs even when they are enqueued via other adapters
+      class Retry
+        def call(worker, msg, queue)
+          msg['retry'] = true unless msg['retry']
```

Comment:
Don't merge this yet. I'm going to try something now that I made a change in the initializer in metrics.